lib/lotus/generators/model/model.rb.tt

```diff
@@ -0,0 +1 @@
+# temporary file
```

Comment:
I want to commit a empty folder `model`. But it can not, So it will raise an error when generate a model :worried:.